### ISSUE TITLE
testing: enable Flaky Bot

### ISSUE THREAD

**Issue by tbpg** [GoogleCloudPlatform/dotnet-docs-samples#964]
Instructions here: https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop

**Comment by tbpg** [comment]
I installed the bot on the repo (first step of the instructions). We still need to call the `buildcop` binary to publish the test logs.

**Comment by amanda-tarafa** [comment]
I'll get to this once I've fixed some already identified issues because of a version bump that we had to take last week, don't want Build cop to flood the repo.

**Comment by tbpg** [comment]
Thanks! Let me know if you run into any issues. :)

**Comment by amanda-tarafa** [comment]
Blocked by #988

**Comment by amanda-tarafa** [comment]
This is now unblocked, I'll get to it this week, right after a pressing issue that came up.

**Comment by meredithslota** [comment]
@amanda-tarafa is this something that @Rishabh-V and team can handle?

**Comment by amanda-tarafa** [comment]
Yep, @Rishabh-V assigning to you, but let's have a chat to give you context before you do anything.

**Comment by Rishabh-V** [comment]
Sure Amanda. I will sync up with you before starting on this one. Thanks.

**Comment by tbpg** [comment]
Updated instructions link: https://github.com/googleapis/repo-automation-bots/tree/main/packages/flakybot

**Comment by amanda-tarafa** [comment]
Closing as obsolete.
